src/Controller/Organization/MembersController.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Buddy\Repman\Controller\Organization;
+
+use Buddy\Repman\Entity\User;
+use Buddy\Repman\Form\Type\Organization\InviteMemberType;
+use Buddy\Repman\Message\Organization\Member\AcceptInvitation;
+use Buddy\Repman\Message\Organization\Member\InviteUser;
+use Buddy\Repman\Message\Organization\Member\RemoveInvitation;
+use Buddy\Repman\Message\Organization\Member\RemoveMember;
+use Buddy\Repman\Query\Admin\Model\User as UserReadModel;
+use Buddy\Repman\Query\User\Model\Organization;
+use Buddy\Repman\Query\User\Model\Organization\Invitation;
+use Buddy\Repman\Query\User\OrganizationQuery;
+use Ramsey\Uuid\Uuid;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;
+use Symfony\Component\Security\Core\Exception\AuthenticationException;
+
+final class MembersController extends AbstractController
+{
+    private OrganizationQuery $organizations;
+    private TokenStorageInterface $tokenStorage;
+
+    public function __construct(OrganizationQuery $organizations, TokenStorageInterface $tokenStorage)
+    {
+        $this->organizations = $organizations;
+        $this->tokenStorage = $tokenStorage;
+    }
+
+    /**
+     * @Route("/organization/{organization}/member", name="organization_members", methods={"GET"}, requirements={"organization"="%organization_pattern%"})
+     */
+    public function listMembers(Organization $organization, Request $request): Response
+    {
+        return $this->render('organization/member/members.html.twig', [
+            'organization' => $organization,
+            'members' => $this->organizations->findAllMembers($organization->id(), 20, (int) $request->get('offset', 0)),
+            'count' => $this->organizations->membersCount($organization->id()),
+            'invitations' => $this->organizations->invitationsCount($organization->id()),
+        ]);
+    }
+
+    /**
+     * @Route("/user/invitation/{token}", name="organization_accept_invitation", methods={"GET"}, requirements={"token"="%uuid_pattern%"})
+     */
+    public function acceptInvitation(string $token): Response
+    {
+        /** @var User $user */
+        $user = $this->getUser();
+        $organization = $this->organizations->getByInvitation($token, $user->getEmail());
+        if ($organization->isEmpty()) {
+            $this->addFlash('danger', 'Invitation not found or belongs to different user');
```

Comment:
This also applies to users without Repman account. Maybe we will change message slightly?

Reply:
Nope, `/user` is protected endpoint and you must be logged in to see it